### ISSUE TITLE
TEMPLATE -- [Article] Some short description of the article

### ISSUE THREAD

**Issue by jawache** [Green-Software-Foundation/writers#59]
## Article Details

**Proposer**: (who is proposing this article idea, Name | Affiliation | GitHub Username)
**Abstract**: (50 word summary of the article)
**Why**: (Why is this article an important topic to discuss and how is it related to the work/mission/theory of change of the GSF)
**Audience**: (Which audience is most going to connect to the content in this article)
**Timeline**: (Should this article come out before or after an important date)

## Important Information

**Author**: (Who is assigned to write this article)
**Other contributors**: (Experts, Reviewers)
**Article Link**: (Link to the doc, must be in our official GSF drive)
**Graphics Link**: (Link to the design assets for this article, must be in our official GSF drive)
**Published Link**: (The link the article was published to for memory)

## Checklist
- [ ] Idea discussed and selected for writing. _After checking this box, set Status to "Todo" under "Artilces" project_
- [ ] Author identified. _After checking this box Assign issue to the Author_
- [ ] Google doc link created in GSF drive. 
- [ ] In progress. _After checking this box, set Status to "In Progress" under "Artilces" project_
- [ ] Draft created. _After checking this box, set Status to "In Review" under "Artilces" project. Assign issue to Reviewer/Author during the back-and-forth_
- [ ] Draft reviewed. _After checking this box, set Status to "In Publishing" under "Artilces" project. Assign issue to Graphic Designer and add "Graphic Design" to issue's Projects_
- [ ] Graphics created. _After checking this box, assign to TBC for approval._
- [ ] Approved. _After checking this box, assign to TBC for publishing._
- [ ] Published. After checking this box, set Status to "Ready" under "Artilces" project. Assign to Marketing and add "D: Marketing" to issue's Projects
- [ ] Shared on socials
- [ ] Shared on newsletter. After checking this box, set Status to "Done" under "Artilces" project and close issue.

**Comment by zanete** [comment]
@jawache @NAMRATA-WOKE @ursvill 

I’ve updated the checklist with some more instructions on how to get from one checkbox to the next. It’s a proposal and probably incorrect / inefficient in some places, please review and let me know your thoughts.

As I was doing this, a few questions arose about the process:
- is it reasonable to expect that a reviewer will be identified at the start of the article (Along with the author)
- would the reviewer be - someone who reviews for style to ensure tone of voice is consistent across articles or a subject matter expert that checks the facts etc?
- How does draft review happen - who should be assigned to drive this forward, e.g. the reviewer reviews and then the author needs to make changes, then the reviewer reviews again etc, would we expect them to change the assignee field every time? Usually people implement some but not all suggestions, so it’s not very clear what the most helpful way would be
- Graphic design - we need a few assets here, don’t we? The article cover image, anything for promotion on socials, too? I think we need to specify that somewhere
- Publishing: Who would do the actual publishing of the article on the website? Would it always need to be Steph? or should be the Author? Or someone on this call?

Once we clarify this, I can proceed to implementing the correct Statuses in the articles project and reflect that workflow on the board

**Comment by jawache** [comment]
> * is it reasonable to expect that a reviewer will be identified at the start of the article (Along with the author)

- I suggested the proposer is also (at least one of) the reviewer as well.

> * would the reviewer be - someone who reviews for style to ensure tone of voice is consistent across articles or a subject matter expert that checks the facts etc?

- I think it depends on who the original author is, if it's someone who is already a subject matter expert then the reviewer should probably be Urs or Namrata.
- If it's an article from Urs then maybe a subject matter expert should review.
- But I also think it totally depends on the context.

> * How does draft review happen - who should be assigned to drive this forward, e.g. the reviewer reviews and then the author needs to make changes, then the reviewer reviews again etc, would we expect them to change the assignee field every time? Usually people implement some but not all suggestions, so it’s not very clear what the most helpful way would be

Once it's a Google doc the conversation tends to move to Google doc comments, I think it has to all happen in there? There is a back and forth that happens on a fairly granular level. Maybe if the communication has stopped we assign it to the person who we are waiting on and add the blocked label to nudge them along.

> * Graphic design - we need a few assets here, don’t we? The article cover image, anything for promotion on socials, too? I think we need to specify that somewhere

Agreed, it should be one package of assets - cover plus socials so it's all covered.

> * Publishing: Who would do the actual publishing of the article on the website? Would it always need to be Steph? or should be the Author? Or someone on this call?

I really don't like that there is this step with a whole other human being to just upload the article, it should be Urs/Namrata, it's pretty straight forward. Fewer humans involved the fewer things that need to be explained and the fewer things that can go wrong.

**Comment by jawache** [comment]
Also FYI I've added this as a template here: https://github.com/Green-Software-Foundation/writers/blob/main/.github/ISSUE_TEMPLATE/new-article-proposal.md

And you can see it when you choose a new issue here: https://github.com/Green-Software-Foundation/writers/issues/new/choose

**Comment by zanete** [comment]
@jawache @NAMRATA-WOKE @ursvill please review the process: https://www.figma.com/file/OD7oDFD2ohMeudtgErFCTA/Find-Repo-for-an-initiative?node-id=0-1&t=4xJObvfD3GCdSSBg-0
and then we can add this template as a form

**Comment by ursvill** [comment]
Looks ok to me. Not sure what I will use it for, though. If it helps your internal processes, then great.
